extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -382,6 +382,14 @@ export class AmpIframe extends AMP.BaseElement {
       return;
     }
 
+    if (height < 100) {
+      user.warn(TAG_,
+          'ignoring embed-size request because the resize height is ' +
+          'less than 100px',
+          this.element);
```

Comment:
We use `user.warn` for resize request on a non-resizable amp-iframe. So I used the same log level. @cramforce What do you think?